### DISCUSSION TITLE
ESP32 config usable in ESP8266

### DISCUSSION THREAD

**Discussion by vibejamin** [arendst/Tasmota#20025] [Q&A] [answered]
Hello everyone,

Out of necessity I used an ESP32 in a project which I'd finally like to replace with the originally intended ESP8266 due to lower power consumption and heat generation.

Before bricking my device I'd like to ask if it is possible to export the 32's Tasmota configuration and import it into the 8266?

Thanks for any help!

cf

**Comment by barbudor** [comment, marked as answer]
Not as a binary DMP file. The DMP files are different for ESP8266 and ESP32. 

However you can use the command line tool [decode-config](https://github.com/Tasmota/decode-config) to get that config as a json file or as a list of commands that you should be able to re-apply back to the new device

**Reply by vibejamin** [reply]
Thanks, I''ll take a look at that tool.
